### PR TITLE
Add web console in debugging rails applications guide [ci skip]  

### PR DESCRIPTION
**Pull request by adidoit**
Added a small section about the Rails Web Console. I find it very useful ! 

First time committer so please help me if this request is wrong in anyway 

thanks !

### CODE REVIEW

File: guides/source/debugging_rails_applications.md [hunk 1]
```diff
@@ -242,6 +242,33 @@ The contents of the block, and therefore the string interpolation, is only
...
+# posts_controller.rb
+class PostsController < ApplicationController
+  def new
+    console
+    @post = Post.new
+  end
+end
+```
+
+By default the web console can only be accessed from localhost.
```
Comment (andreynering) [inline guides/source/debugging_rails_applications.md]: What should I do if I want to access it from another device?

File: guides/source/debugging_rails_applications.md [hunk 2]
```diff
@@ -242,6 +242,33 @@ The contents of the block, and therefore the string interpolation, is only
...
+console is launched automatically in case on an error but can also be launched for debugging purposes
+by inserting `<%= console %>`anywhere in a view or `console` anywhere in a controller method.
+
+For example:
+
+```ruby
+# new.html.erb
+<%= console %>
+```
+
```
Comment (andreynering) [inline guides/source/debugging_rails_applications.md]: What about an "or" between these code blocks?

Comment (adipra3n) [inline guides/source/debugging_rails_applications.md]: Thanks. I pushed a178573 to make the language clearer with an 'Or' and also added details about configuring network access.

Comment (andreynering) [inline guides/source/debugging_rails_applications.md]: @adipra3n Usually, it's a good idea to keep a single commit per PR. If they decide to merge, they will ask you to squash the commits first.

Comment (adipra3n) [inline guides/source/debugging_rails_applications.md]: ah yes, makes sense @andreynering. I squashed the commits into ad8ca74 now

**Comment by gsamokovarov** [comment]
Thank you @adipra3n for taking the initiative on this!

File: guides/source/debugging_rails_applications.md [hunk 3]
```diff
@@ -242,6 +242,44 @@ The contents of the block, and therefore the string interpolation, is only
 evaluated if debug is enabled. This performance savings is only really
 noticeable with large amounts of logging, but it's a good practice to employ.
 
+
+Debugging with the Web Console
```
Comment (gsamokovarov) [inline guides/source/debugging_rails_applications.md]: Should we say:

```markdown
Debugging with the `web-console` gem
------------------------------------
```

so we are consistent with byebug heading below?

File: guides/source/debugging_rails_applications.md [hunk 4]
```diff
@@ -242,6 +242,44 @@ The contents of the block, and therefore the string interpolation, is only
...
+```
+
+By default the web console can only be accessed from localhost. `config.web_console.whitelisted_ips` 
+lets you control which IPs have access to the console. For example, to allow access from both localhost
+and 192.168.0.100:
+
+```ruby
+# config/application.rb
+class Application < Rails::Application
+  config.web_console.whitelisted_ips = %w( 127.0.0.1 192.168.0.100 )
```
Comment (gsamokovarov) [inline guides/source/debugging_rails_applications.md]: Maybe its worth noting that you can whitelist a whole network as well:

```ruby
class Application < Rails::Application
  config.web_console.whitelisted_ips = %w( 127.0.0.1 192.168.0.0/16 )
end
```
Also, that outside of the [private networks](http://en.wikipedia.org/wiki/Private_network), this is pretty insecure :)

Comment (adipra3n) [inline guides/source/debugging_rails_applications.md]: hey @gsamokovarov I edited to add your suggestions -  b7d316c

thanks for this excellent gem :)

**Pull request merged**